cli/src/infragpt/llm/__init__.py

```
@@ -10,23 +10,23 @@
 __version__ = "0.1.0"
 
 # Public exports from client.py
-from llm.client import (
+from infragpt.llm.client import (
     get_llm_client,
```

Ruff (0.11.9)

`infragpt.llm.client.get_llm_client` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)